database/migrations/2019_04_03_141031_php_script_limit.php

```diff
@@ -18,7 +18,7 @@ public function up()
 
```

Comment:
This file should not be modified.
While it will work for any new user, it will not pass to the users who already have the latest step installed.
The correct approach is to create a new migration:
`php artisan make:migration rename_php_script_limit`
and add in it something like:
```php
	    Configs::where('key','php_script_limit')->update([
			    'key' => 'php_script_no_limit'
	    ]);
```
Take example on https://github.com/LycheeOrg/Lychee-Laravel/blob/master/database/migrations/2019_02_19_193500_rename_justified_layout.php

Reply:
done.